### ISSUE TITLE
New Design required for simulator

### ISSUE THREAD

**Issue by sjathin** [bhatvineeth/SchedulingSimulation#31]
**Is your feature request related to a problem? Please describe.**
The current design of the simulator does not have the best look and feel. And includes a lot of third-party dependencies on the UI (which are unnecessary). 

The current UI:
![Screenshot 2021-05-30 at 19 41 13](https://user-images.githubusercontent.com/29340368/120114338-042d0a00-c17f-11eb-81a8-75f75094efae.png)

**Describe the solution you'd like**
A completely new design. This does not include the coding aspect, a few designs as to how the simulator UI should be.  

**Additional context**
The design need not be too complicated and with rich features for starting. A basic design is required so that we can move the front end from Django to a much reliable front-end component.

**Comment by bhatvineeth** [comment]
Preferably keeping the charts as it is. (The representation of data using those specific charts)

**Comment by sjathin** [comment]
Yes, thanks for reminding me, the graph will be generated by chart.js, so the design for the graph can be as it is. If their needs to be a change, it must adhrere to the chart.js framework.
